server/lib/utils.js

```diff
@@ -193,6 +193,16 @@ export const defaultHostId = () => {
 };
 
 /**
+ * Demo host id, set this for new groups created through the flow
+ */
+export const demoHostId = () => {
+  if (process.env.NODE_ENV === 'production' || process.env.NODE_ENV  === 'staging') {
+    return 254;
```

Comment:
Just noting this kind of hard-coding will break the day another DB instance would be spun up